sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -233,11 +268,13 @@ def encoded_items(self):
 
 
 class FnApiRunner(runner.PipelineRunner):
+  _num_workers = 1
```

Comment:
This should be an instance, not class-level, variable. Class variables are global state.